eZ/Bundle/EzPublishIOBundle/BinaryStreamResponse.php

```diff
@@ -133,6 +137,11 @@ public function setContentDisposition( $disposition, $filename = '', $filenameFa
      */
     public function prepare( Request $request )
     {
+        if ( !$this->file instanceof BinaryFile )
```

Comment:
It's a bit old for me now :p.
The error occured for me on the prepare method if I remember well so I started to had check on it.

I had checking on setFile methode too : setAutoLastModified and setContentDisposition use $this->file property.

More over even if setFile checks $this-file property, in my optinion, prepare method use directly $this->file and so should check again $this->file (imagine deleted file for any reason).

As you whish :).